example_sns_init.yaml

```diff
@@ -207,39 +220,46 @@ Voting:
         # Values of 0 result in the reward rate always being `final`.
         transition_duration: 12 years
 
-# This structure sets the initial token distribution of the SNS. You can configure how SNS tokens
-# are distributed in three "buckets": tokens that are given to the original developers of
-# the dapp, treasury tokens that are owned by the SNS governance canister,
-# and swap tokens which are swapped in a decentralization swap.
+# Configuration of the initial token distribution of the SNS. You can configure
+# how SNS tokens are distributed in each of the three groups:
+# (1) tokens that are given to the original developers of the dapp,
+# (2) treasury tokens that are owned by the SNS governance canister, and
+# (3) tokens which are distributed to the decentralization swap participants.
 #
-# All developer tokens are distributed to the defined principals at genesis in a basket
-# of neurons called developer neurons.
+# TODO[ATG]: what is "The Swap's initial balance"?
 #
-# The initial token distribution must satisfy the following preconditions to be valid:
-#    - The sum of all developer tokens in E8s of a token must be less than u64::MAX
-#    - The Swap's initial balance must be greater than 0
-#    - The Swap's initial balance must be greater than or equal to the sum of all developer tokens
+# The initial token distribution must satisfy the following preconditions to be
+# valid:
+#    - The sum of all developer tokens in E8s must be less than `u64::MAX`.
+#    - The Swap's initial balance must be greater than 0.
+#    - The Swap's initial balance must be greater than or equal to the sum of
+#      all developer tokens.
 Distribution:
-    # The neurons distributed at genesis to the development team of the dapp being
-    # decentralized. A Neuron specifies the neuron's stake, controlling principal,
-    # a unique memo, a dissolve delay, and a vesting period. Even though these
-    # Neurons are distributed at genesis, they are locked in PreInitializationMode
-    # until the decentralization swap is completed.
+    # TODO[ATG]: how is unique memo practically used? Is the uniqueness actually 
+    # enforced?
```

Comment:
Its used in the NeuronId calculation, which is special-hash(principal_id, memo). If developers want multiple neuron's controlled by their principal, they will need to use unique memos.